website/blog/2024-08-18-introduce-tabby-inline-edit/index.mdx

```diff
@@ -0,0 +1,94 @@
+---
+authors: [wei]
+tags: [editor integration, product]
+image: ./tabby-edit.png
+
+---
+
+
+# Introduce the Inline Edit in Tabby
+
+![Tabby Edit](./tabby-edit.png)
+
+## What is Tabby Edit?  
+  
+Tabby Edit is a feature that allows you to edit your code directly in your favorite editor.  
+  
+Tabby Edit will not change your code directly,  
+but it will show you the difference between your code and the generated ones,  
+allow you to review the changes and decide whether to apply.  
+  
+  
+## Get started with Tabby Edit  
+  
+To get started with Tabby Edit, you need a Tabby server and extension installed for your editor.  
+If you don't have it done, please check [our documents](https://tabby.tabbyml.com/docs/welcome/).  
+After you have installed the Tabby server and extension, you can start using Tabby Edit.  
+  
+First, select the code you want to update and then use the command palette to open Tabby Edit,  
+or use the keyboard shortcut `Ctrl + I` (or `Cmd + I` on Mac).  
+  
+Then you can type your prompt to describe what you want to do,  
+and Tabby will generate the code for you showing the difference between your code and the generated ones.  
+  
+![Tabby Edit](./tabby-edit.png)  
```

Comment:
This image is confusing to me. Maybe a GIF to showcase the full experience as you are describing - select code, command palette, prompt, choose Accept or Discard

Reply:
This image only served as a header image, GIFs are appended later in the blog, should we also post a GIF here?